sonic-xcvrd/scripts/xcvrd

```diff
@@ -0,0 +1,430 @@
+#!/usr/bin/env python
+
+"""
+    Xcvrd
+    Transceiver information update daemon for SONiC
+"""
+
+try:
+    import getopt
+    import os
+    import imp
+    import signal
+    import subprocess
+    import sys
+    import syslog
+    import time
+    import threading
+    from swsscommon import swsscommon
+except ImportError, e:
+    raise ImportError (str(e) + " - required module not found")
+
+#============================= Constants =============================
+
+VERSION = '1.0'
+
+SYSLOG_IDENTIFIER = "xcvrd"
```

Comment:
>xcvrd [](start = 21, length = 5)

get the script basename programatically?

Reply:
+1. This is copied from other daemons I've written, and I've also been meaning to go back over them and obtain the basename programmatically.

`SYSLOG_IDENTIFIER = os.path.basename(__file__)` should do the trick.